data/changelog.txt

```diff
@@ -147,438 +150,97 @@ Add reminder effects for some medicine.
-Makeshift knapsack.
-Human skin and stomachs.
 More railroad terrain variants, made diagonal tracks subway railroads 7 tiles wide.
 Implement surrounded start.
-Expands variety of items in NPC NC itemgroups.
-Add bone fishing hook and bone skewer.
 Professions: Hunters, Bandit, Bionic Survivalist, Parkour Practitioner, Burglar, Camper, Road Warrior, Boxer, Photojournalist, Tourist, Zookeeper.
-The Red Dragon Teashop.
-Hotel improvements (JSON, new lobbies, updated item, monster, and vehicle spawns).
-Lab and mutant start: Adds ears and tails to traits
-Mix condensed milk and water for 2 milk.
-Adds Cinnamon.
-Add Optical Dampers CBM.
-Add chai tea.
-Convert bullwhip to use reach attack.
-Football Field.
-Scorpion451's XL Holster
-Add recipe for chocpretzels and choco_coffee_beans
-Wearable binoculars V2 Updated
-Make zombies drop tainted leather
-Soylent green
+New mapgen: The Red Dragon Teashop, Football Field.
 Double the number of survivor's notes.
-French toast
-Add pumpkin and sunflower as farmable plants
-Leather/bone/chitin recipe staggering, armguard split
-add tortillas
-Crossbow hunter
-Zombie child evolution expansion.
-New build_lab, other underground overmap code cleanup
-Adding eggnog
-Tin can sealer.
-Use 4 hotdogs instead of one when making cooked hot dogs.
-Add "calories" field to it_comest
-Unscrewable vehicle parts
-Inferior Weapons Patch
-Buildable Reinforced glass + Metal shutters.
-Quesadillas!
-Adding a pepperbox flintlock
-Adding pepperbox pistol to mansion loot list
-Terrain connections for groups other than WALL
-Hands free mechanics
-Add more makeshift firearms
-Add new vehicle parts
-More canning recipes.
-Adds electronic earmuffs that reduce sounds to a safe level.
-Add basic chemistry set
-Mainline glaive from Medieval Stuff mod
-Move longsword from Medieval Stuff mod
-Shield Foundation Part 2: BLOCK_WHILE_WORN
-Make inwoods runs less bland
-Add metal funnel
-Overmap scent traces
-Move halberd from Medieval Stuff mod
-Adds deconstruction recipe for rope and pulley.
-Put butcher knife in the weapons category
-Mutant wildlife expansions: Spiders and snakes
-Shield foundation part 1: RESTRICT_HANDS
-Shield Foundation Part 3: Bringing it all together
-Add ammo pouches
+Add "calories" field to it_comest.
+Terrain connections for groups other than WALL.
+Hands free mechanics.
 Update Evac Center.
-Neoprene expansion
-Add more metal constructions
-Rivtech magazine update.
-Added magazine to Military Recruit background
-Makes electrohack battery-powered
-Mark spare magazine as obsolete
-Make bio_ups an active bionic
-Implement MAG_DESTROY and MAG_EJECT
-Add recipes for some armor items
-Add missing cotton material to neoprene armsleeves.
-Add missing magazines to professions
-Add more magazine spawns
-Adds Kel-Tech SUB-2000
-Adds Saiga-12 magazines.
-Additional Sandwich Types
-New vehicle: Automatic Tractor
-Cannabis enhancement.
+Implement MAG_DESTROY and MAG_EJECT.
```

Comment:
I'm not against this, but I also don't know what it is and it seems weird to call it out.

Reply:
Agreed, looked the actual change up and it's essentially a patch on the whole gun magazine addition effort.